src/lib.rs

```diff
@@ -20,7 +20,7 @@
 //
 // This means that missing docs will still fail CI, but means we can use
 // README.md as the crate documentation.
-#![cfg_attr(feature = "nightly", deny(missing_docs))]
+#![cfg_attr(all(feature = "nightly"), deny(missing_docs))]
```

Comment:
This `all` seems unnecessary?

Reply:
Yeah, it's a leftover from some temporary flags that I used.